paymentsheet/src/main/java/com/stripe/android/common/ui/ElementsBottomSheetLayout.kt

```diff
@@ -56,7 +57,8 @@ internal fun ElementsBottomSheetLayout(
     StripeBottomSheetLayout(
         state = state,
         layoutInfo = layoutInfo,
-        modifier = modifier,
+        modifier = modifier
+            .imePadding(),
```

Comment:
_hoisted_ imePadding a bit up so that users can customize it, and adds it for the Elements bottom sheet to keep current behavior.